### PR TITLE
Refactor manager start and creation of stores

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

File: cmd/onos-config/onos-config.go [hunk 1]
```diff
@@ -78,59 +67,31 @@ func main() {
...
-		log.Fatal("Cannot load device atomix store ", err)
-	}
-
-	networkChangesStore, err := network.NewAtomixStore(atomixClient)
-	if err != nil {
-		log.Fatal("Cannot load network atomix store ", err)
-	}
-
-	networkSnapshotStore, err := networksnap.NewAtomixStore(atomixClient)
+	modelRegistry, err := modelregistry.NewModelRegistry(modelregistry.Config{})
```
Comment (adibrastegarnia) [inline cmd/onos-config/onos-config.go:70]: we should be able to move this to manager as well correct?

**Review by adibrastegarnia — CHANGES_REQUESTED** [review]

File: cmd/onos-config/onos-config.go (hunk 1 above)
Comment (ray-milkey) [reply cmd/onos-config/onos-config.go:70]: Can't move this unfortunately. Model registry must be initialized before the manager is created. The tests have a special implementation of this.

Comment (ray-milkey) [reply cmd/onos-config/onos-config.go:70]: hmm no. I think I can move it now. Let me try

File: cmd/onos-config/onos-config.go [hunk 2]
```diff
@@ -78,59 +66,25 @@ func main() {
...
-		log.Fatal("Cannot load network atomix store ", err)
-	}
-
-	deviceStateStore, err := state.NewStore(networkChangesStore, deviceSnapshotStore)
-	if err != nil {
-		log.Fatal("Cannot load device store with address %s:", *topoEndpoint, err)
-	}
-	log.Infof("Topology service connected with endpoint %s", *topoEndpoint)
+	mgr := manager.NewManager(cfg)
+	log.Info("Manager created")
```
Comment (adibrastegarnia) [inline cmd/onos-config/onos-config.go:79]: Perhaps we can remove this log as well and just log it inside Run function

**Review by adibrastegarnia — APPROVED** [review]

**Pull request merged**